instruqt/distributing-with-replicated/track.yml

```diff
@@ -1,5 +1,5 @@
 slug: distributing-with-replicated
-id: evmpzv6ycpgs
+id: 
```

Comment:
blank id

Reply:
I think the id's are blanked out by the dev/prod workflow